static/js/components/dashboard/courses/Grades_test.js

```diff
@@ -123,4 +129,10 @@ describe('Course Grades', () => {
     course.has_exam = false;
     assert.equal(renderGrades().find('.passed-course').length, 0);
   });
+
+  it('should call setShowGradeDetailDialog onClick', () => {
```

Comment:
Can you also test that if you click it when it's visible it calls `setShowGradeDetailDialog` with `false?